dali/test/python/operator_1/test_numba_func.py

```diff
@@ -140,37 +174,69 @@ def test_numba_func():
           np.full((10, 30, 20), 42, dtype=np.int32)]),
     ]
 
+    device = "cpu"
     for shape, dtype, run_fn, out_types, in_types, outs_ndim, ins_ndim, \
             setup_fn, batch_processing, expected_out in args:
         yield _testimpl_numba_func, \
-            shape, dtype, run_fn, out_types, in_types, outs_ndim, ins_ndim, \
+            device, shape, dtype, run_fn, out_types, in_types, outs_ndim, ins_ndim, \
             setup_fn, batch_processing, expected_out
 
 
+def test_numba_func_gpu():
+    # shape, dtype, run_fn, out_types,
+    # in_types, out_ndim, in_ndim, setup_fn, batch_processing,
+    # expected_out
+    args = [
+        ([(10, 10, 10)], np.uint8, set_all_values_to_255_sample_gpu, [dali_types.UINT8],
+         [dali_types.UINT8], [3], [3], None, None,
+         [np.full((10, 10, 10), 255, dtype=np.uint8)]),
+        ([(10, 10, 10)], np.float32, set_all_values_to_float_sample_gpu, [dali_types.FLOAT],
+         [dali_types.FLOAT], [3], [3], None, None,
+         [np.full((10, 10, 10), 0.5, dtype=np.float32)]),
+        ([(100, 20, 30), (20, 100, 30)], np.int64, change_out_shape_sample_gpu, [dali_types.INT64],
+         [dali_types.INT64], [3], [3], setup_change_out_shape, None,
+         [np.full((20, 30, 100), 42, dtype=np.int32),
+          np.full((100, 30, 20), 42, dtype=np.int32)]),
+    ]
+
```

Comment:
Can I ask for some variability in the dimensionality of the tested data?

Reply:
I added a test that gets 1D input and  4D output